man/git-sed.md

```diff
@@ -3,12 +3,12 @@ git-sed(1) -- replace patterns in git-controlled files
 
 ## SYNOPSIS
 
-`git-sed` [ -c ] [ -f <flags> ] <search> <replacement>
+`git-sed` [ -c ] [ -f <flags> ] <search> <replacement> [ <flags> ]
```

Comment:
Also need to update the html and man page.
See https://github.com/tj/git-extras/blob/master/man/Readme.md